assets/src/blocks/amp-story-text/get-tag-name.js

```diff
@@ -1,12 +1,25 @@
+/**
+ * WordPress dependencies
+ */
+import { count } from '@wordpress/wordcount';
+import { _x } from '@wordpress/i18n';
+
 // Todo: Make these customizable?
 const H1_FONT_SIZE = 40;
 const H2_FONT_SIZE = 24;
+const H1_TEXT_LENGTH = 15;
+const H2_TEXT_LENGTH = 30;
+
+/*
+ * translators: If your word count is based on single characters (e.g. East Asian characters),
+ * enter 'characters_excluding_spaces' or 'characters_including_spaces'. Otherwise, enter 'words'.
+ * Do not translate into your own language.
+ */
+const wordCountType = _x( 'words', 'Word count type. Do not translate!', 'amp' );
```

Comment:
Are the `H1_TEXT_LENGTH` and `H2_TEXT_LENGTH` values definitely meant in words? 15 words seems quite long for a `h1` on the AMP Story Page :)

Reply:
Ugh, I guess I was thinking in characters.